scalafmt-core/shared/src/main/scala/org/scalafmt/config/Newlines.scala

```diff
@@ -113,6 +113,9 @@ case class Newlines(
     avoidAfterYield: Boolean = true
 ) {
   val reader: ConfDecoder[Newlines] = generic.deriveDecoder(this).noTypos
+
+  def beforeSingleArgParenLambdaParams(implicit edition: Edition): Boolean =
```

Comment:
Let's make this a val in `ScalafmtConfig` and remove the implicit edition variable.

Reply:
will do